compiler/passes/errorHandling.cpp

```diff
@@ -20,14 +20,55 @@
 #include "errorHandling.h"
 
 #include "AstVisitorTraverse.h"
+#include "CatchStmt.h"
 #include "stmt.h"
 #include "symbol.h"
 #include "TryStmt.h"
-#include "view.h"
 
 #include <stack>
 #include <vector>
 
+/*
```

Comment:
I like to see such listings showing what the pass is supposed to do. However, this comment needs a sentence or two of introduction since it might seem out of the blue to a reader. It also needs to mark what the input and output of the pass are.